src/ui/components.tsx

```diff
@@ -44,6 +45,7 @@ interface CheckMkGenericAsyncSelectProps<Value> extends CommonProps<Value> {
   width?: number;
   autocompleter: (prefix: string) => Promise<Array<SelectableValue<NonNullable<Value>>>>;
   inputId: string; // make the InlineField magic do its work // TODO: find better solution
+  showVariables?: boolean;
```

Comment:
I'm not sure about an optional boolean prop, at least `true | undefined` is better, if typescript can do boolean constant type like it can with strings.

Otherwise, we should document a default value.

Reply:
setting it to `true | undefined` make this property non optional, so you would have to explicitly set it to undefined. then i guess a normal boolean would be better.
i changed it to `showVariables?: false` and added a comment.